libraries/RW/SocialScrape/SocialScrape.py

```diff
@@ -0,0 +1,53 @@
+# Inspired from https://github.com/MartinBeckUT/TwitterScraper/blob/master/snscrape/python-wrapper/snscrape-python-wrapper.py
+# # Medium Article Follow-Along: https://medium.com/better-programming/how-to-scrape-tweets-with-snscrape-90124ed006af
+"""
+SocialScrape keyword library
+Based on snscrape https://github.com/JustAnotherArchivist/snscrape
+ 
+Scope: Global
+"""
+
+import requests
+import urllib
+import json
+import dateutil.parser
+
+from datetime import timedelta, date
+import datetime
+from RW import platform
+import snscrape.modules.twitter as sntwitter
+import pandas as pd
+
+class SocialScrape: 
+    """
+    Twitter Scraper keyword library
+    Uses https://github.com/JustAnotherArchivist/snscrape
+    """
+
```

Comment:
We typically add an explicit:
`ROBOT_LIBRARY_SCOPE = "GLOBAL"`
in class stanzas to indicate their scoping within robot suites. This indicates to robotframework to maintain 1 instance of the class within the suite. It won't make a functional difference to what you've written but it's a good maintainability addition.